pkg/apiserver/plugins.go

```diff
@@ -83,4 +86,31 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	shootstatedeletionvalidator.Register(plugins)
 	customverbauthorizer.Register(plugins)
 	resourcequota.Register(plugins)
+	shootvpa.Register(plugins)
+}
+
+// DefaultOffAdmissionPlugins gets admission plugins off by default for gardener-apiserver.
+func DefaultOffAdmissionPlugins() sets.String {
+	defaultOnPlugins := sets.NewString(
+		lifecycle.PluginName,                       // NamespaceLifecycle
+		resourcereferencemanager.PluginName,        // ResourceReferenceManager
+		extensionvalidation.PluginName,             // ExtensionValidator
+		shoottolerationrestriction.PluginName,      // ShootTolerationRestriction
+		shootdns.PluginName,                        // ShootDNS
+		shootquotavalidator.PluginName,             // ShootQuotaValidator
+		shootvalidator.PluginName,                  // ShootValidator
+		seedvalidator.PluginName,                   // SeedValidator
+		controllerregistrationresources.PluginName, // ControllerRegistrationResources
+		plantvalidator.PluginName,                  // PlantValidator
+		deletionconfirmation.PluginName,            // DeletionConfirmation
+		openidconnectpreset.PluginName,             // OpenIDConnectPreset
+		clusteropenidconnectpreset.PluginName,      // ClusterOpenIDConnectPreset
+		shootstatedeletionvalidator.PluginName,     // ShootStateDeletionValidator
+		customverbauthorizer.PluginName,            // ShootVPA
+		mutatingwebhook.PluginName,                 // MutatingAdmissionWebhook
+		validatingwebhook.PluginName,               // ValidatingAdmissionWebhook
+		resourcequota.PluginName,                   // ResourceQuota
+	)
```

Comment:
Can you extract this into an `DefaultOnAdmissionPlugins` or `DefaultEnabledAdmissionPlugins` function?
It's going to be more intuitive for devs to change an `DefaultOnAdmissionPlugins` instead of `DefaultOffAdmissionPlugins` (due to negation), if a (new) plugin should be enabled by default.

Reply:
Yeah, good suggestion.
I extracted two vars `DefaultOnPlugins` and `DefaultOffPlugins` to make it more intuitive to devs.
PTAL.

Reply:
Nice, thank you 👍